src/common/utils/reduxStoreUtilty.js

```diff
@@ -0,0 +1,12 @@
+import { createStore } from "redux";
+import { setDataReduxSate } from "../../common/store/commonReduxStoreReducer";
+
+export const store = createStore(setDataReduxSate);
+
+export const storeDispachObservations = (type, value) => {
```

Comment:
Do we need this wrapper over store.dispatch? We can just import the store directly where we want to dispatch an action and do ```store.dispatch(types.OBSERVATIONS_VALUE, observation)```